src/client/pfs/pfs.proto

```diff
@@ -513,14 +513,14 @@ message FileInfoV2 {
 //     Client sends a request with the EOF field set to true.
 //   Client connection is closed.
 
-message PutTarRequestV2 {
+message PutTarRequest {
```

Comment:
It seems like it would be better to keep the V2 suffix for all messages / rpcs specific to Pachyderm 2. Although we don't need it for name collisions, it seems like it would be a good design to have clear lines between the V1 and V2 functionality.